src/components/testCaseList/TestCaseList.tsx

```diff
@@ -111,12 +112,20 @@ const TestCaseList = () => {
                 );
 
                 if (groupPopulation) {
-                  groupPopulation.actual = populationResult.result;
                   executionStatus =
                     groupPopulation.expected === populationResult.result;
                 }
               }
             });
+            if (executionStatus && !!stratificationValues) {
+              stratificationValues.forEach((stratVal) => {
+                if (executionStatus) {
```

Comment:
Is this check required, it looks like at this point, the executionStatus will always be true.